client/app/bundles/KudosApp/components/Kudo.jsx

```diff
@@ -1,127 +1,175 @@
+        onClick={ likedBySelf ? unlikeKudo(id) : likeKudo(id) }
+      />
+    )
+  }
+
+  renderEditOptions() {
     const Edit = () => (
-      <button
-        type='button'
-        className="kudo__edit-button"
-        onClick={this.makeEditable}>
-        Edit
-      </button>
+      <i
+        className="far fa-edit"
+        onClick={ this.makeEditable }
+      />
     )
 
     const Save = () => (
-      <button
-        type='button'
-        className='kudo__edit-button kudo__edit-button--save'
-        onClick={this.update}>
-        Save
-      </button>
+      <i
+        className="far fa-save"
+        onClick={ this.update }
+      />
     )
 
-    const recipientDisplay = map(this.props.kudo.receivers, 'name').join(', ')
+    const Delete = () => (
+      <i className="far fa-trash-alt" onClick={ ()=> alert("delete")}></i>
+    )
+
+    // TODO: enable Delete button when implemented in backend
     return (
-      <div className="kudo">
-        <h4 className="list-group-item-heading">{`Kudos, ${recipientDisplay}!`}</h4>
-        <div className="kudo__receiver">
-          <img src={this.props.kudo.receivers[0].avatar} alt={this.props.kudo.receivers[0].name} className="kudo__avatar" />
+      this.postedByActiveUser() ? (
+        <div>
+          { this.state.editing ? <Save /> : <Edit /> }
+          {/* <Delete /> */}
         </div>
-        <div className="kudo__message">
-          <blockquote className="blockquote">
-            {this.state.editing ? (
-              <textarea
-                className="kudo__input"
-                rows={3}
-                value={this.state.body}
-                onChange={this.setMessage}
-              />
-            ) : (
-              this.state.body
-            )}
-            <footer className="blockquote-footer">{this.props.kudo.giver}</footer>
-          </blockquote>
+      ) : null
+    )
+  }
+
+  render() {
+    return (
+      <div className="kudo">
+        <div className="content">
+          <div className="sender">
+            <UserAvatar user={ this.props.kudo.giver } />
+          </div>
+          <div className={ "receiver " + this.props.color_class }>
+            <div className="header">
+              { this.formattedHeaderText() }
+              { this.renderRecipientAvatars() }
+            </div>
+            <div className="message">
+              { this.renderBody() }
+            </div>
+          </div>
         </div>
-        <FloatingActionButton onClick={this.state.likeAction} mini={true} backgroundColor={this.state.thumbColor}>
-          <ThumbUp/>
-        </FloatingActionButton>
-        <div>{this.state.likeText}</div>
-        {this.postedByActiveUser() ? (
-          <div className="kudo__update">
-            {this.state.editing ? <Save /> : <Edit />}
+        <div className="meta">
+          <div>
+            { this.renderLikeIcon() }
+            { this.formattedLikeText() }
```

Comment:
Let's just remove the extra like copy altogether.

Reply:
Do you mean remove the text following the number of likes? e.g. "1 person likes this"

Reply:
Yeah, I'm thinking only numbers is fine, what do you think?